include/soci/db2/soci-db2.h

```diff
@@ -201,7 +201,7 @@ struct SOCI_DB2_DECL db2_statement_backend : details::statement_backend
 
     int prepare_for_describe();
     void describe_column(int colNum, data_type& dtype, std::string& columnName);
-    std::size_t column_size(int col);
+    SQLUINTEGER column_size(int col);
```

Comment:
This makes it inconsistent with the ODBC and Oracle versions, should the other 2 be changed as well? If not, I'd rather leave this one alone as well and keep the casts, in SQL/ODBC code they are admittedly unavoidable sometimes...